tests/playbooks/kubevirt_pvc_img_upload.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Create a PVC from local VM image
+  hosts: localhost
+  connection: local
+  tasks:
+    - name: Create a PVC annotated for upload
+      # This will be shorter with kubevirt_pvc
+      k8s:
+       state: present
+       namespace: default
+       name: pvc-vm1
+       inline:
+         apiVersion: v1
+         kind: PersistentVolumeClaim
+         metadata:
+           name: pvc-vm1
+           labels:
+             app: containerized-data-importer
+           annotations:
+             cdi.kubevirt.io/storage.upload.target: ""
```

Comment:
Should that be target PVC?

Reply:
That's just the weird way CDI makes you annotate a PVC ready for image upload. In `kubevirt_pvc` I was planning on masking it with something like `upload: bool`. https://github.com/kubevirt/containerized-data-importer/blob/master/doc/upload.md#create-a-persistentvolumeclaim